src/view/Command.java

```diff
@@ -0,0 +1,7 @@
+package view;
+
+import java.io.IOException;
+
+public interface Command {
+    String doCommand(String arg) throws ClassNotFoundException, IOException;
```

Comment:
What will happen for multiple args?
Maybe change to Arraylist<String>?

Reply:
Definitely make sense, but - according to the assignment, your interface class, and the logic (only one string argument) - I think it should not be changed.

Reply:
ok, know for future stuff that if not explicitly defined - flexible solution wins (even more than the simple 1 arg one)